packages/clay-tooltip/src/TooltipProvider.tsx

```diff
@@ -66,6 +66,20 @@ const reducer = (state: IState, {type, ...payload}: IAction): IState => {
 	}
 };
 
+function closestParent(node: HTMLElement, s: string) {
+	const el = node;
+	let ancestor: HTMLElement | null = node;
+
+	if (!document.documentElement.contains(el)) return null;
+
+	do {
+		if (ancestor.matches(s)) return ancestor;
```

Comment:
I thought it was our convention to  always wrap statements in curlies. eg.

```javascript
if (ancestor.matches(s)) {
    return ancestor;
}

Reply:
my bad, I had copied this code from a polyfil and forgot to format it.